app/controllers/AnnotationController.scala

```diff
@@ -225,6 +225,26 @@ class AnnotationController @Inject()(
       } yield result
     }
 
+  @ApiOperation(hidden = true, value = "")
+  def removeAnnotationLayer(typ: String, id: String, layerName: String): Action[AnyContent] =
+    sil.SecuredAction.async { implicit request =>
+      for {
+        _ <- bool2Fox(AnnotationType.Explorational.toString == typ) ?~> "annotation.makeHybrid.explorationalsOnly"
+        annotation <- provider.provideAnnotation(typ, id, request.identity)
```

Comment:
```suggestion
        annotation <- provider.provideAnnotation(typ, id, request.identity)
        _ <- bool2Fox(annotation._user == request.identity._id) ?~> "notAllowed" ~> FORBIDDEN
```

Reply:
It is clear that there needs to be some authorization, but is it really intended that only the creator of the annotation can edit it?

Reply:
Fair question! The answer is not entirely clear, but yes, I’d build it like this for now.

There are two other groups to consider
 - people who “administrate” this user (organization admins and team managers): here the usual workflow is that they are *not* allowed to edit the annotation at all. Instead they should use “copy to my account” if they want to create a copy to modify themselves. So not allowing them to delete a layer is in line with the existing behavior.
 - people who the annotation was shared with if `othersMayEdit` was set to true by the annotation owner: here we could also allow them to delete layers but I think since it is a potentially drastic destructive action it would be safer not to allow this at this time. If people who are using the othersMayEdit feature request this in the future, we may still change this.

Reply:
:+1: